src/MangaStreamParser.ts

```diff
@@ -253,7 +252,7 @@ export class MangaStreamParser {
         for (const manga of mangas.toArray()) {
             const title = section.titleSelectorFunc($, manga)
 
-            const image = this.getImageSrc($('img', manga)) ?? ''
+            const image = this.getImageSrc($('img', manga), baseUrl) ?? ''
```

Comment:
What's the purpose for this and why should it be added? Has not been needed before?

Reply:
> Some mangastream sites use src attributes like "https://google.com/images/abc.png" and others '/images/abc.png".
I thought it might be a nice feature to have an already implemented system might we add some of these mangastream sources in the future.